week0001/fizzbuzz_daudau.php

```diff
@@ -0,0 +1,4 @@
+<?php
+	$i = 1;
+	while ($i++ < 100)
+		echo $i%15 ? $i%3 ? $i%5 ? '' : 'buzz'.'<br/>' : 'fizz'.'<br/>' : 'fizz'.'<br/>'.'buzz'.'<br/>';
```

Comment:
I can see how this works: Nested ternary operators replace explicit blocks of `if/else`. There are 3 issues with this solution:

1. Even when `$i` is not a multiple of 3 or 5, you still call `echo` function and force it to print a blank! That's a waste of 47 function calls!
2. Would you understand this code 2 months from now?
3. Would other people understand & maintain your code without refactoring & changing it? Save them effort.